mycoffee/functions.py

```diff
@@ -89,20 +86,39 @@ def load_params(args):
     return params
 
 
-def coffee_calc(params, digits=3):
+def filter_params(params, digits=3):
     """
-    Calculate coffee.
+    Filter params.
 
     :param params: parameters
     :type params: dict
     :param digits: number of digits up to which the given number is to be rounded
     :type digits: int
+    :return: filtered parameters as dict
+    """
+    params["coffee"] = round(params["coffee"], digits)
+    if is_int(params["coffee"]):
+        params["coffee"] = int(params["coffee"])
+    if is_int(params["water_ratio"]):
+        params["water_ratio"] = int(params["water_ratio"])
+    if is_int(params["coffee_ratio"]):
+        params["coffee_ratio"] = int(params["coffee_ratio"])
+    if is_int(params["water"]):
+        params["water"] = int(params["water"])
```

Comment:
Why do you have `round` only for `coffee`. Just making sure there is no mistake there.

Reply:
We only round the output, while the other parameters are input.